Automattic/jetpack
# Improve Jetpack products illustrations file size

Fixes https://github.com/Automattic/jetpack/pull/21497#pullrequestreview-787069943

#### Changes proposed in this Pull Request:

* This PR improves the product illustration images by running them through TinyPNG smart loss compression (no visual change/loss). The goal is to reduce the file size.

#### Jetpack product discussion


Context:  https://github.com/Automattic/jetpack/pull/21497#pullrequestreview-787069943

#### Does this pull request change what data or activity we track or use?


Nope

#### Testing instructions:






* Make sure the new images are not blurry
* Go to the following pages and confirm the images don't have any visual changes:
  * `/wp-admin/admin.php?page=jetpack#/product/akismet`
  * `/wp-admin/admin.php?page=jetpack#/product/backup`
  * `/wp-admin/admin.php?page=jetpack#/product/scan`
  * `/wp-admin/admin.php?page=jetpack#/product/search`
  * `/wp-admin/admin.php?page=jetpack#/product/security`
